app/legacy/entitlements.js

```diff
@@ -2,7 +2,7 @@ const { get } = require('../api')
 
 const getEntitlements = async (organisationId, callerId) => {
   const url = `/SitiAgriApi/entitlements/grouped/${organisationId}`
-  const data = await get(url, callerId)
+  const data = await get(url, callerId) ?? 0
```

Comment:
I don't think this will work as if data is undefined it will set to 0.  But then you'll try to get `0.data` on the next line